src/mina-signer/MinaSigner.ts

```diff
@@ -493,6 +493,15 @@ class Client {
     let sk = PrivateKey.fromBase58(privateKeyBase58);
     return createNullifier(message, sk);
   }
+
+  /**
+   * Returns the network ID.
+   * 
+   * @returns {NetworkId} The network ID.
+   */
+  getNetworkId(): NetworkId {
```

Comment:
nit: this doesn't seem like a useful method to me since you can do `this.network`?

if you added this to use the "network id" terminology instead of "network", I suggest to make this a getter, so we can change its behaviour back to a plain property access later:

```
get networkId(): NetworkId
```